layers/+lang/d/packages.el

```diff
@@ -14,6 +14,7 @@
 (setq d-packages
       '(
         company
+        company-dcd
```

Comment:
Add `:toggle (configuration-layer/package-usedp 'company)` here and it looks good to me.